mainwindow.cpp

```diff
@@ -278,19 +289,13 @@ MainWindow::~MainWindow()
 void MainWindow::on_pushButton_WordlistFileBrowse_clicked()
 {
     // We pop a dialog to choose a file to open.
-    // TODO: *.lst for file extension?
-    // TODO: Copy-pasting is evil! (open password file)
-    // TODO: What happens when John writes something while dialog
-    //       opened?
-    // TODO: Move dialog creation and setting up into window constructor.
     QFileDialog dialog;
     dialog.setFileMode(QFileDialog::ExistingFile);
-    // TODO: Dialog could allow user to select multiple files. May it
-    //       be good to support this ability? To concatenate selected file?
```

Comment:
On Thu, May 14, 2015 at 11:00:48AM -0700, math07 wrote:
> >      QFileDialog dialog;
> >      dialog.setFileMode(QFileDialog::ExistingFile);
> > -    // TODO: Dialog could allow user to select multiple files. May it
> > -    //       be good to support this ability? To concatenate selected file?
> 
> Is it something that you guys would really want ?

john can load from multiple files in one call. Concatenation is not
needed. Just `john t1.pw t2.pw t3.pw` works.

Thanks!

-- 
Regards,
Aleksey Cherepanov